client/my-sites/plans/plan-overview/plan-remove/index.jsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Dialog from 'components/dialog';
+import { isInGracePeriod } from 'lib/plans';
+import paths from '../../paths';
+
+const PlanRemove = React.createClass( {
+	propTypes: {
+		plan: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired
+	},
+
+	getInitialState() {
+		return {
+			showDialog: false
+		};
+	},
+
+	closeDialog( action ) {
+		this.setState( { showDialog: false } );
+
+		if ( action === 'remove' ) {
+			// TODO: Add action that calls the API and cancels the free trial
+
+			// TODO: Display success notice
+
+			page( paths.list( this.props.selectedSite.slug ) );
+		}
+	},
+
+	showDialog( event ) {
+		event.preventDefault();
+
+		this.setState( { showDialog: true } );
+	},
+
+	renderCard() {
+		return (
+			<CompactCard className="plan-remove">
+				<strong>{ this.translate( 'Not looking to purchase?' ) }</strong>
```

Comment:
In general it's not a bad idea to split the sentences if they are quite independent. Mostly it's a judgement call, though. In this case I'd suggest to put the two into one `translate()` (because they belong quite closely together) but leave the Remove link separate.